llvm/test/ObjectYAML/DXContainer/PSVv2-compute.yaml

```diff
@@ -70,18 +70,18 @@ Parts:
 # CHECK-NEXT: NumThreadsZ:     2048
 # CHECK-NEXT: ResourceStride: 24
 # CHECK-NEXT: Resources:
-# CHECK-NEXT: - Type:            1
+# CHECK-NEXT: - Type:            Sampler
 # CHECK-NEXT: Space:           2
 # CHECK-NEXT: LowerBound:      3
 # CHECK-NEXT: UpperBound:      4
-# CHECK-NEXT: Kind:            5
-# CHECK-NEXT: Flags:           6
-# CHECK-NEXT: - Type:            128
+# CHECK-NEXT: Kind:            TextureCube
+# CHECK-NEXT: Flags:           0
+# CHECK-NEXT: - Type:            Invalid
```

Comment:
Why did "Flags" change from `6` to `0`? Also shouldn't a value of `0` for flags print "None" since that's the name of the value in the ResourceFlag enum?

Reply:
Flags need to be able express FlagA | FlagB, so it has to be uint. But we only have 0 and 1 now, so maybe make it enum is OK?

Reply:
Ah, fair. Using the actual value for flags seems fine. Is that a heuristic in the yaml printer or something? I don't see any code that explicitly opts in or out of the enum value printing.

Reply:
Oh I see. We don't actually use the ResourceFlag enum in this change at all, so `LLVM_YAML_DECLARE_ENUM_TRAITS(llvm::dxbc::PSV::ResourceFlag)` and `ScalarEnumerationTraits<dxbc::PSV::ResourceFlag>::enumeration` are just entirely dead code.

Reply:
You're right.
I'll remove these.

Reply:
Removed in https://github.com/llvm/llvm-project/pull/106617